tests/303_mongodb_import_connect_cli.robot

```diff
@@ -5,29 +5,30 @@ Metadata            Version    0.0.1
 Resource            ../resources/keywords/deploy_application.resource
 
 Suite Setup         Run Keywords
-...                 Set Library Search Order    OpenShiftLibrary
-...                 AND    Skip If    ${DBaaSPolicyEnabled}
+...                     Set Library Search Order    OpenShiftLibrary
+...                     AND    Skip If    ${DBaaSPolicyEnabled}
 Suite Teardown      Tear Down The Test Suite
 Test Setup          Given Login To OpenShift CLI
 Test Teardown       Tear Down The Test Case
-Force Tags          CLI      mongo
+
+Force Tags          CLI    mongo
 
 
 *** Test Cases ***
 Scenario: Verify Error Message For Invalid Credentials On MongoDB Using OC CLI
-    [Tags]    smoke    RHOD-470
+    [Tags]    RHODA-015    smoke
     When User Creates MongoDB Secret With Invalid Credentials
     And User Imports MongoDB Provider Account Using CLI
     Then Provider Account Import Failure Using CLI
 
 Scenario: Import MongoDB Provider Account Using OC CLI
-    [Tags]    smoke    RHOD-460
+    [Tags]    RHODA-016
```

Comment:
Why isn't this smoke too?

Reply:
The flow is covered as part of "Scenario: Deploy MongoDB Provider Account Using OC CLI"